src/sharing/components/ContactSuggestion.jsx

```diff
@@ -32,12 +31,17 @@ export const ContactSuggestion = ({ contactOrGroup, contacts, t }) => {
     })
     avatarText = 'G'
   } else {
-    const email = Contact.getPrimaryEmail(contactOrGroup)
-    name = email || getDisplayName(contactOrGroup)
-    avatarText =
-      Contact.getInitials(email) || Contact.getInitials(contactOrGroup)
+    if (contactOrGroup.fullname) {
+      name = contactOrGroup.fullname
+      avatarText = Contact.getInitials(contactOrGroup)
+    } else {
+      name = Contact.getPrimaryEmail(contactOrGroup)
+      avatarText = name[0].toUpperCase()
```

Comment:
'Passing a string to Contact.getInitials will be deprecated soon.' :D

Reply:
yeah, that's why I use this :-)

Reply:
Is it possible to have a contactOrGroup without email? Should we test if name[0] exists ?